lib/diaspora/shareable.rb

```diff
@@ -77,11 +77,15 @@ def from_person_visible_by_user(user, person)
       end
 
       def for_visible_shareable_sql(max_time, order, limit=15, types=Stream::Base::TYPES_OF_POST_IN_STREAM)
-        by_max_time(max_time, order).order(table_name + ".id DESC").where(type: types).limit(limit)
+        by_min_max_time(Date.parse('01-01-2008'), max_time, order).order(table_name + ".id DESC").where(type: types).limit(limit)
```

Comment:
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Metrics/LineLength: Line is too long. [129/120]